tests/periph_uart/tests/periph_uart_if.py

```diff
@@ -17,7 +17,10 @@ class PeriphUartIf(DutShell):
 
     def uart_init(self, dev=DEFAULT_DEV, baud=DEFAULT_BAUD):
         """Initialize DUT's UART."""
-        return self.send_cmd("init {} {}".format(dev, baud))
+        ret = self.send_cmd("init {} {}".format(dev, baud))
+        # Clear buffer from init by sending and recieving
+        self.send_cmd("send {} {}".format(dev, "junk"))
```

Comment:
`junk` sound a bit rough, maybe `drop` or `flush` is more descriptive

Reply:
Thanks, done!